support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -261,11 +262,22 @@ function deriveSubsAcquisitionData(
 
 }
 
+function deserialiseAusMomentAcquisitionData(serialised: string): Object {
+  const [page, socialPlatform, referralCode] = serialised.split('_');
+  return {
+    componentId: `${page}_${socialPlatform}`,
+    source: 'SOCIAL',
+    campaignCode: referralCode,
```

Comment:
I've just realised an issue here. Elsewhere it overrides the `campaignCode` if the user has arrived on a campaign page. In this case that would be the /au/contribute contributions page.

Reply:
I think we have 2 options for the referralCode here:
1. Include it in `componentId`, which would then just be the full `<page>_<platform>_<code>`
2. Add it to the `queryParameters` field - https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/helpers/tracking/acquisitions.js#L46

@paulbrown1982 any thoughts on what's better from an analysis perspective?

Reply:
@tomrf1 @paulbrown1982 has a decision been reached about how to solve this problem?

Reply:
The referralCode will need to be query-able, for the emailing.
To me having it in `queryParameters` seems more robust than sticking it on the end of the `componentId` string.
@paulbrown1982 what do you think?

Reply:
The referralCode / shareCode should be inside `queryParameters`, and not go anywhere near `campaignCode`.

Reply:
The `componentId` would be the call to action that the user clicked on to arrive at this pageview. In this case, it would be the Twitter link, so that should probably be `<page>_<platform>`.

Reply:
Ok @tomrf1 @paulbrown1982 , I've moved the referralCode into queryParams, see the updated PR description for more.